viewer/packages/rendering/src/rendering/EffectRenderManager.ts

```diff
@@ -913,8 +913,11 @@ function createRenderTarget(
   options?: THREE.WebGLRenderTargetOptions
 ): THREE.WebGLRenderTarget {
   if (multiSampleCountHint > 1) {
-    const rt = new THREE.WebGLMultisampleRenderTarget(1, 1, { ...options, ignoreDepth: false } as any);
-    rt.samples = multiSampleCountHint;
+    const rt = new THREE.WebGLRenderTarget(1, 1, {
+      ...options,
+      ignoreDepth: false,
```

Comment:
I don't think this is a thing? Or is it new?

Reply:
Have removed this option